recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -12,6 +12,16 @@ add_task(withDriver(Assert, async function uuids(driver) {
   isnot(uuid1, uuid2, "uuids are unique");
 }));
 
+add_task(withDriver(Assert, async function installXpi(driver) {
+  // Test that we can install an XPI from any URL
+  const dir = getChromeDir(getResolvedURI(gTestPath));
+  dir.append("fixtures");
+  dir.append("normandy.xpi");
```

Comment:
@Mossop normandy.xpi was indeed unsigned.. I've updated the xpi so that it's signed now.